homework1/task2/check_fib/check_fib.py

```diff
@@ -0,0 +1,24 @@
+from math import sqrt
+from typing import Sequence
+
+
+def is_fibonacci(element) -> bool:
+    element_1 = 5 * element ** 2 - 4
+    element_2 = 5 * element ** 2 + 4
+    if element == 0:
+        return True
+    else:
+        if sqrt(element_1).is_integer() or sqrt(element_2).is_integer():
+            return True
+        else:
+            return False
```

Comment:
тут можно упростить:
```suggestion
    else:
        return sqrt(element_1).is_integer() or sqrt(element_2).is_integer()
```